### PR TITLE
Update next from master

### PR DESCRIPTION
**Pull request by awharn**
**What It Does**
Ports changes from #1842 to `next`

**How to Test**

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [ ] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by traeok — APPROVED** [review]
LGTM, thanks @awharn!

File: jest.config.js
Comment (t1m0thyj) [inline jest.config.js:132]: Curious about the reason for this change? When the Jest Stare results are stored in a top-level folder, the coverage link breaks. I think we'd want to keep the old directory as the default value, and allow the `.env` files to override it.

Comment (awharn) [reply jest.config.js:132]: I'm going to move this into draft mode and wait for my other PR to be merged before merging that in here and addressing this.

Comment (awharn) [reply jest.config.js:132]: Addressed

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**
